Hero-overlay-forest/hero.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forest</title>
+    <link rel="Stylesheet" href="hero.css" />
+  </head>
+  <body>
+    <div class="forest"></div>
+    <div class="darkness"></div>
```

Comment:
Aici as spune sa pui asa
```suggestion
    <div class="forest">
     <div class="darkness"></div>
    </div>
```